### ISSUE TITLE
http://yadcf-showcase.appspot.com/entrys_table_source?_=1674682280312 giving back Error 500

### ISSUE THREAD

**Issue by Ciao121** [vedmack/yadcf#672]
Hi,
I was trying to understand ajax source and looking at http://yadcf-showcase.appspot.com/ajax_source.html the ajax request fails with error 500 so the page is not working.

**Comment by vedmack** [comment]
you are right, untill it get resolved, you can take a loot at the source code of the showcase in my github repo https://github.com/vedmack/yadcf-showcase

**Comment by vedmack** [comment]
not sure when it will be resolved, seem that google app engine started to block these request cause I haven't activated billing on my showcase :|
